apps/ensrainbow/src/lib/routes.ts

```diff
@@ -0,0 +1,37 @@
+import type { EnsRainbow } from "@ensnode/ensrainbow-sdk";
+import { Hono } from "hono";
+import type { Context as HonoContext } from "hono";
+import { logger } from "../utils/logger";
+import { ENSRainbowDB } from "./database";
+import { ENSRainbowServer } from "./server";
+
+/**
+ * Creates and configures the ENS Rainbow server routes
+ */
+export async function createRoutes(db: ENSRainbowDB): Promise<Hono> {
+  const app = new Hono();
+  const rainbow = await ENSRainbowServer.init(db);
```

Comment:
Perhaps we could align all factory methods to be either `init` or `create` to align these current calls:
- `ENSRainbowDB.create`
- `ENSRainbowServer.init`

Reply:
For `ENSRainbowDB` we have to methods: `create` and `open` which create and init this object. Maybe we can add 
```
init(create: bool):
  if create: this.create()
  else: this.open()
```